File: binaryTree/java/Node.java
package binaryTree.java;

class Node {
    int data;

    Node left;
    Node right;

    public Node(int data) {
        this.data = data;

        this.left = null;
        this.right = null;
    }

    @Override
    public String toString() {
        return "Node [data=" + data + ", left=" + left + ", right=" + right + "]";
    }

    public static void main(String[] args) {
        Node root = new Node(1);

        root.left = new Node(2);
        root.right = new Node(3);

        root.right.left = new Node(4);

        System.out.println(root);

    }
}

File: binaryTree/javascript/Node.js
function Node(data, left, right) {
    this.data = data
    this.left = left === undefined ? null : left;
    this.right = right === undefined ? null : right;
}

const binaryTree = new Node(1, new Node(2, new Node(3), new Node(4)), new Node(5, null, new Node(6)))

console.log(binaryTree)